source/options-storage.ts

```diff
@@ -26,7 +26,7 @@ const migrations = [
 	featureWasRenamed('remove-unused-repo-tabs', 'clean-repo-tabs'), // Merged in July
 	featureWasRenamed('more-dropdown', 'clean-repo-tabs'), // Merged in July
 	featureWasRenamed('remove-diff-signs', 'hide-diff-signs'), // Merged in August
-	featureWasRenamed('remove-label-faster', 'hide-label-faster'), // Merged in August
+	featureWasRenamed('remove-label-faster', 'quick-label-hiding'), // Merged in August
 
```

Comment:
Please add migration for all renamed features

Reply:
Including a migration from hide-label-faster to quick-label-hiding

Reply:
I don't think it's needed it was never released. WDYT?